misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathFeatureAssertionsSpec.kt

```diff
@@ -51,12 +52,12 @@ abstract class PathFeatureAssertionsSpec(
             it("toBe(folder.parent) holds") {
                 val childFolder = tempFolder.newFolder("child")
                 val parentFolder = childFolder.parent
-                expect(childFolder).parentVal().toBe(parentFolder)
+                toBe(parentFolder,expect(childFolder))
```

Comment:
don't use `AnyAssertionsBuilder.toBe` it was perfectly fine how it was previously. I guess there is a misunderstanding in the goal of the task. You need to change how `tempFolder` is specified and used.

Reply:
So i was getting errors on both tempFolder and toBe. For tempFolder i could see there is another class MemoizedTempFolder. But it does not have a method perTest() or anything similar to that.

Reply:
That's correct, there isn't a perTest or perGroup anymore, this is control via CachingMode of memoized. See https://github.com/robstoll/tutteli-spek-extensions#MemoizedTempFolder for an example how it is used.